01 - Community/People/🗂️ People.md

```diff
@@ -513,7 +513,7 @@ Overviews of all the work from notable People like Plugins Developers, Theme Des
 -  [[01 - Community/People/ZaherAlMajed|ZaherAlMajed]]
 -  [[01 - Community/People/zapthedingbat|zapthedingbat]]
 -  [[01 - Community/People/zcysxy|zcysxy]]
--  [[01 - Community/People/Zektor|Zektor]]
+-  [[01 - Community/People/Zektor|Hugo Santos (Zektor)]]
```

Comment:
This will get overwritten by a script, that's why it's fenced off with "Do not edit" comments.

Reply:
sorry about that